doc/classes/ProjectSettings.xml

```diff
@@ -1051,6 +1051,9 @@
 		<member name="physics/2d/cell_size" type="int" setter="" getter="" default="128">
 			Cell size used for the broad-phase 2D hash grid algorithm (in pixels).
 		</member>
+		<member name="physics/2d/debug/draw_collision_outlines" type="bool" setter="" getter="" default="true">
```

Comment:
Maybe better suited in `debug/shapes/collision/` that in a new sub-category?

Reply:
Ah yes, thanks!

Reply:
Done.